app/posts/register-trainee-teachers/2020-09-07-register-teacher-trainees-alpha-prototype.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Initial alpha static prototype 
```

Comment:
If the project is in beta now, this could be "Alpha prototype"

Reply:
Another good shout. Will update next pass.